bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -85,7 +85,7 @@
               </ul>
             </div>
             {% if l10n_has_tag('firefox_new_other_platforms') %}
```

Comment:
> I'm saying that we can drop the l10n_has_tag only here, since the string will be translated, not in in the whole page.

And I'm saying we can't drop it here alone, since it would cause the page to break ;)